### DISCUSSION TITLE
Hosting Hubs without Trusting Amazon

### DISCUSSION THREAD

**Discussion by martindale** [mozilla/hubs#3287] [Hubs Cloud Questions] [answered]
It's not clear how to run the "server" for Hubs ("Hubs Cloud") on local hardware, i.e., without [trusting a third-party service](https://nakamotoinstitute.org/trusted-third-parties/).

Maybe I missed something, but can documentation / links be provided for hosting the coordinator(s) on my own hardware?

**Comment by robinkwilson** [comment, marked as answer]
Great question! Though a duplicate question. Search for `self hosting` on discussions to pull these up. Hope these help!

Relevant discussion threads:
https://github.com/mozilla/hubs/discussions/3099
https://github.com/mozilla/hubs/discussions/2699
https://github.com/mozilla/hubs/discussions/3049
